Shuttlerock/shuttlerock_shared_config
# [Suttlerock_shared_config] Add auto publishing to npm

## Add auto publishing to npm

[Trello Card](https://trello.com/c/GstgzlIP/14215-suttlerocksharedconfig-add-auto-publishing-to-npm)

Need to add auto publishing to npm(like to rubygems)


## How to test the PR

- Add changes to project 
- Update gem's version (lib/shuttlerock_shared_config/version.rb ; package.json)
- push changes to github 
- Add tag with new version and push to git, example
```
git tag -a v0.2.9 -m "v0.2.9"
git push origin v0.2.9
```
- check versions shuttlerock_shareg_config on RubyGems and NPM

## Deployment Notes

no
